packages/button/src/button.css

```diff
@@ -21,3 +21,72 @@ governing permissions and limitations under the License.
         border-color: highlight;
     }
 }
+
+@keyframes show-progress-circle {
+    0% {
+        visibility: hidden;
+    }
+
+    100% {
+        visibility: visible;
+    }
+}
+
+@keyframes hide-icons-label {
+    0% {
+        visibility: visible;
+    }
+
+    100% {
+        visibility: hidden;
+    }
+}
+
+@keyframes update-button-styles {
```

Comment:
Let's name this something about `pending`, so it's more clear what it is for.

Reply:
done